WaterDock-2.0_Basics.md

```diff
@@ -0,0 +1,76 @@
+**This file is a more detailed version of README.md.**
+N.B. The command line commands in this text file work for Mac but may not work for Windows and Linux. You might need to look up the equivalent commands for these operating systems.
+
+HOW TO SET UP WATERDOCK-2.0 (DETAILED)
+
+1.   Download the zip file or clone using GitHub.
+
+
+2.   Have Python 3.7 (or the latest version of Python 3 installed onto your computer).
+
+
+3.   Ensure that you have the MDAnalysis, numpy and scipy packages installed using either pip or conda.
+      - Look up the correct commands for installing each package using Google\
+
+
+4.   Install Autodock Vina using this website "http://vina.scripps.edu/download.html" to any folder that you like.
+     - It installs as a zip file, so open it to export the folder into the same directory as the zip file.
+
+
+5.   Add Autodock Vina program (found within the bin folder of the folder you have downloaded) to your PATH.
+      - You can view your PATH using the command `echo $PATH` in Terminal.
+      - To add to your path, use this command `sudo nano /etc/paths`.
+      - Type in the password for your computer.
+      - Press the down key until you reach the bottom of the text, which should include other directories like `/usr/bin`. 
+      - Type in the directory that should end in `.../vina/bin`.
+      - Press Control-X.
+      - Press Y and then Enter.
+      - Restart the Terminal.
+
+
+6.   Navigate to your WaterDock-2.0 folder using the `cd` command and its directory and run the example.
+      - Use the command `python3 waterdock2.py example-protein.pdbqt example-ligand.pdb`.
+      - This produces a file called **predicted waters.pdb** in the folder which can be opened with Pymol or just with a Text editor. This should contain 3 waters.
+
+
+If you get that result, you have WaterDock-2.0 working!
+
+
+
+
+HOW TO RUN WATERDOCK-2.0 ON YOUR OWN PROTEIN-LIGAND COMPLEXES
+
+1.   Download the protein-ligand complex pdf file from the Protein Databank  (https://www.rcsb.org/)
+     - 1g9v is being used as an example for the command line commands.
+
+
+2.   Separate the protein from the protein complex to produce a pdb file that only contains the protein using this command line command (when in its directory).
+      - `more 1g9v.pdb |grep ATOM > 1g9v_prot.pdb`

```

Comment:
The `more` call here seems unnecessary, can you not just do `grep ATOM 1g9v.pdb > 1g9v_prot.pdb`?

Reply:
Yes you can, I did not know that. I have changed it.